daheinze-oscheer/go.sum

```diff
@@ -0,0 +1,31 @@
+github.com/kisielk/errcheck v1.4.0 h1:ueN6QYA+c7eDQo7ebpNdYR8mUJZThiGz9PEoJEMGPzA=
```

Comment:
You don't have to push this file

Reply:
you are right. but it was installed based on the tutorial. i remove it.